src/data/tags.ts

```diff
@@ -1,3 +1,29 @@
-export const tags = ['JavaScript', 'React', 'Vue', 'Java', 'C#', '.Net'] as const;
+export const tags = [
+  'JavaScript',
+  'TypeScript',
+  'Node.js',
+  'NestJs',
+  'React',
+  'Vue',
+  'Java',
+  'C#',
+  '.Net',
+  'SRE',
+  'DevOps',
+  'IT Operations',
+  'Scalability',
+  'Software Developer',
+  'Software Engineer',
+  'Project Management',
+  'Software Planning',
```

Comment:
What is Software Planning? Is this mentioned somewhere?

Reply:
I just added related tags. We can remove it. this is no big deal.

Reply:
I don't know what is this tag. Please remove it.